### PR TITLE
Feature/ivory 396 confirmation page

### PR DESCRIPTION
**Pull request by Jozzey**
Added new dynamic content to page.
Created new unit tests.
Updated packages and version number

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ivory",
-  "version": "0.10.1",
+  "version": "0.10.2",
```
Comment (duncannixon) [inline package.json:3]: This should be a 'minor' increment, so 0.11.0, as this is new functionality rather than a bug fix or rebuild.

**Review by duncannixon — CHANGES_REQUESTED** [review]
Minor change to the version number, otherwise all looks good.

File: package.json (hunk 1 above)
Comment (Jozzey) [reply package.json]: 👍🏼

**Review by duncannixon — APPROVED** [review]
Good work

**Pull request merged**